demo-deploy/README.md

```diff
@@ -0,0 +1,53 @@
+# Infrastructure Deployment Makefile
+
+This repository includes a `Makefile` to streamline the deployment, management, and teardown of AWS infrastructure using Terragrunt. The script ensures all prerequisites are met and simplifies executing commands for planning, applying, and destroying infrastructure across all modules in a specific environment.
+After creating the infrastructure, which includes the ECR repository, you can use the push-deploy Github workflow to deploy the Zebra Server to ECR and the ECS cluster.
```

Comment:
List all infrastructure commponent here.
Why do we need lambda for this? please mention the reason. if this is for logs, mention that tere is another repo for the code.

Reply:
a proper infrastucture list is in order
- ECR container for Zebra docker files
- ECS to run the Zebra test net
- Lamda for logs
- anything else?